router/core/router.go

```diff
@@ -594,6 +595,21 @@ func (r *Router) newServer(ctx context.Context, routerConfig *nodev1.RouterConfi
 			subChiRouter.Use(metricHandler.Handler)
 		}
 
+		subgraphs := make([]Subgraph, len(routerConfig.Subgraphs))
+		for _, s := range routerConfig.Subgraphs {
+			parsedURL, err := url.Parse(s.RoutingUrl)
+			if err != nil {
+				r.logger.Error("Failed to parse subgraph url", zap.Error(err))
```

Comment:
Please specify the subgraph url that could not be parsed otherwise it is hard to debug.

Reply:
Added